src/app/templates/partials/header.ftl

```diff
@@ -31,11 +31,13 @@
         <a class="dropdown-toggle" href="#" data-toggle="dropdown">
             Logged in as ${user}
         </a>
-        <div class="dropdown-menu">
+        <div class="dropdown-menu dropdown-menu-right">
             <#if isAdmin && fineGrainedAuth>
             <a class="dropdown-item" href="${appUrl}/manage-access">Manage access</a>
             </#if>
+            <#if isReviewer && fineGrainedAuth>
```

Comment:
I think we want this link to be visible if fineGrainedAuth is not available. In that case, all logged in users have the same permissions, so all users are able to publish. (Ah yes, just checked report-page.ftl, and that is the logic it uses for the publish switch on there). so this should be `<#if isReviewer || !fineGrainedAuth>`  This is a bit different to manage access, since changing permissions only makes sense when fineGrainedAuth is enabled anyway. 

Sorry, should have explained this before!